static_src/actions/user_actions.js

```diff
@@ -200,10 +200,25 @@ const userActions = {
 
   clearInviteNotifications() {
     AppDispatcher.handleViewAction({
-      type: userActionTypes.USER_INVITE_STATUS_DISMISSED
+      type: userActionTypes.USER_LIST_NOTICE_DISMISSED
     });
   },
 
+  createUserListNotification(noticeType, description) {
```

Comment:
I think we should start writing new actions in this format:
```
type: RAD_TYPE,
data: {
  datum1: false,
  datum2: true
}
```

Then every action presents a consistent interface regardless of the information being passed in. What do you think?